src/main/java/macrobase/analysis/stats/distribution/Wishart.java

```diff
@@ -0,0 +1,49 @@
+package macrobase.analysis.stats.distribution;
+
+import org.apache.commons.math3.linear.EigenDecomposition;
+import org.apache.commons.math3.linear.RealMatrix;
+import org.apache.commons.math3.special.Gamma;
+
+public class Wishart {
+    private final double logDetOmega;
+    private RealMatrix omega;
+    private double nu;
+    private int D;  // dimensions
+
+    public Wishart(RealMatrix omega, double nu) {
+        this.omega = omega;
+        this.nu = nu;
+        this.D = omega.getColumnDimension();
+        this.logDetOmega = Math.log((new EigenDecomposition(omega)).getDeterminant());
+    }
+
+    public double lnB() {
```

Comment:
Again, comments would be good. No "B" in https://en.wikipedia.org/wiki/Wishart_distribution